src/pages/ManageStaff/ManageStaff.jsx

```diff
@@ -78,6 +82,13 @@
   };
 
   useEffect(() => {
+    const getRole = async () => {
+      const userId = getCurrentUserId();
```

Comment:
🚫 **[eslint]** <[no-shadow](https://eslint.org/docs/rules/no-shadow)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'userId' is already declared in the upper scope on line 42 column 10.